projects/element-ng/schematics/migrations/wizard-migration/index.ts

```diff
@@ -0,0 +1,144 @@
+
+    const tsSourceFiles = await discoverSourceFiles(tree, context, options.path);
+
+    for (const filePath of tsSourceFiles) {
+      const content = tree.read(filePath);
+      if (!content) {
+        continue;
+      }
+
+      const sourceFile = ts.createSourceFile(
+        filePath,
+        content.toString(),
+        ts.ScriptTarget.Latest,
+        true
+      );
+
+      const modulePathToMatch = /@(siemens|simpl)\/element-ng(\/wizard)?/;
+
+      const wizardImports = getImportSpecifiers(sourceFile, modulePathToMatch, [
+        'SiWizardComponent',
+        'SiWizardModule'
+      ]);
+
+      if (!wizardImports?.length) {
+        continue;
+      }
+
+      const recorder: UpdateRecorder = tree.beginUpdate(filePath);
+
+      renameApi({
+        tree,
+        filePath,
+        sourceFile,
+        recorder,
+        elementName: 'si-wizard'
+      });
+
+      if (recorder) {
+        tree.commitUpdate(recorder);
+      }
+    }
+  };
+};
+
+const renameApi = ({
+  tree,
+  filePath,
+  sourceFile,
+  recorder,
+  elementName
+}: {
+  tree: Tree;
+  filePath: string;
+  sourceFile: ts.SourceFile;
+  recorder: UpdateRecorder;
+  elementName: string;
+}): void => {
+  getInlineTemplates(sourceFile).forEach(template =>
+    addOrRemoveInlineNavigationAttribute({
+      template: template.text,
+      offset: template.getStart() + 1,
+      elementName,
+      recorder
+    })
+  );
+  getTemplateUrl(sourceFile).forEach(templateUrl => {
+    const templatePath = join(dirname(filePath), templateUrl);
+    const templateContent = tree.read(templatePath)!.toString('utf-8');
+    const templateRecorder = tree.beginUpdate(templatePath);
+    addOrRemoveInlineNavigationAttribute({
+      template: templateContent,
+      offset: 0,
+      elementName,
+      recorder: templateRecorder
+    });
+    tree.commitUpdate(templateRecorder);
+  });
+};
+
+/**
+ * Migrates the inlineNavigation attribute on wizard elements.
+ * - Adds `inlineNavigation` when not present (new default: true)
+ * - Removes `[inlineNavigation]="false"` (false is now the default)
+ * - Keeps `[inlineNavigation]="true"` and dynamic bindings unchanged
+ */
+
+const addOrRemoveInlineNavigationAttribute = ({
+  template,
+  offset,
+  recorder,
+  elementName
+}: {
+  recorder: UpdateRecorder;
+  template: string;
+  offset: number;
+  elementName: string;
+}): void => {
+  const elements = findElement(template, element => element.name === elementName);
+
+  for (const element of elements) {
+    const inlineNavigationAttr = element.attrs.find(
+      attr => attr.name === '[inlineNavigation]' || attr.name === 'inlineNavigation'
+    );
+
+    if (!inlineNavigationAttr) {
+      // No attribute exists → Add default inlineNavigation
+      const insertPosition = element.startSourceSpan.end.offset + offset - 1;
+      recorder.insertLeft(insertPosition, ' inlineNavigation');
+    } else if (
+      inlineNavigationAttr.name === '[inlineNavigation]' &&
```

Comment:
We have an attribute converter, so it should also work without property binding:


```suggestion
      (inlineNavigationAttr.name === '[inlineNavigation]' || inlineNavigationAttr.name === 'inlineNavigation') &&
```

Reply:
Done